specificatie/domain.yaml

```diff
@@ -590,6 +593,21 @@ components:
                         Dit kan een aangeboden stuk of een kadasterstuk zijn.
           items:
             type: string
+    KadastraalOnroerendeZaakFiliatie:
+      type: "object"
+      description: |
+                    De eigenschappen van de kadastraalOnroerendeZaak die gerelateerd is.
+      properties:
+        identificatie:
+          description: |
+                        De identificatie(s) van de Kadastraal Onroerende Za(a)k(en).
+          type: "string"
+        indicatieVervallen:
+          description: |
+                        Geeft aan of de er verwezen wordt een vervallen kadastraal onroerende zaak.
+          type: "boolean"
+        appartementsrechtVolgnummer:
```

Comment:
hiermee wordt appartementsrechtVolgnummer toegevoegd aan isOntstaanUit en isOvergegaanIn. Dat was niet gevraagd #843. Toevoegen van appartementsrechtVolgnummer aan de onroerende zaak en aan bijbehorendeAppartementsrechten is daarvoor voldoende. Waarom is het hier ook toegevoegd?

Reply:
Ik heb uiteindelijk een samenvatting (https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/issues/843#issuecomment-862342909) gemaakt van wat ik dacht dat er aangepast moest worden. Daar stond de bovenstaande wijziging tussen vanwege deze (https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/issues/843#issuecomment-862286141 ) conclusie. Als ik daar de verkeerde conclusie aan heb verbonden hoor ik dat graag, dan haal ik het appartementsrechtVolgnummer weer uit de Filiatie.

Reply:
Appartementsrechtvolgnummer verwijderd n.a.v. overleg 30-06-2021